src/urbit/app/chess.hoon

```diff
@@ -653,9 +654,15 @@
         ?~  archived-game  ~
         ``[%chess-game !>(u.archived-game)]
       ``[%chess-game !>(game.u.active-game)]
-    ::
-    ::  .^(arch %gy /=chess=/game)
-    ::  collect all the game-id keys
+  ::
+  ::  .^(noun %gx /=chess=/mutuals/noun)
+  ::  .^(json %gx /=chess=/mutuals/json)
+  ::  read mutual pals
+    [%x %mutuals ~]
+      ``[%chess-pals !>((~(mutuals pals bowl) ~.))]
```

Comment:
I'll mention here that the mark is called `chess-pals` rather than `chess-mutuals` because we could, and maybe will, extend that same mark to deal with different calls like `chess-update` does.